src/components/statistics/index.js

```diff
@@ -12,7 +12,7 @@ class Statistics extends Component {
             <Wrapper>
                 <StatisticsWrapper>
                     <Statistic>
-                        <Number>{this.props.totalStakedNodes}</Number>
+                        <Number>5000&plus;</Number>
```

Comment:
is it still firing off the failing call in the background?

Reply:
Oh yeah, totally forgot to remove it. Done in https://github.com/pokt-foundation/explorer/pull/134/commits/44ed7aebebea03e3248b56a9f69728c4bc669c49!